### ISSUE TITLE
Some inconsistent market symbols

### ISSUE THREAD

**Issue by rubobaquero** [ccxt-dev/ccxt#390]
Hi again :-),

I have found some incredible arbitrages. I am sure that all or most of them are due to assigning different coins for the same symbol on different exchanges. I paste them below:

```
ARBITRAGE FOUND: BTC/DOGE => 0.00000019 (bxinth) - 5,263,157.89473684 (ccex) => 5,263,157.89473665 (2,770,083,102,492,974.00%)
ARBITRAGE FOUND: TIX/LTC => 0.00000003 (bter) - 0.00030001 (cryptopia) => 0.00029998 (999,933.33%)
ARBITRAGE FOUND: BTC/LTC => 0.01046799 (bxinth) - 95.65780525 (ccex) => 95.64733726 (913,712.54%)
ARBITRAGE FOUND: NET/BTC => 0.00000010 (cryptopia) - 0.00014750 (liqui) => 0.00014740 (147,400.00%)
ARBITRAGE FOUND: BLC/BTC => 0.00000152 (cryptopia) - 0.00009505 (ccex) => 0.00009353 (6,153.29%)
ARBITRAGE FOUND: BTG/BTC => 0.00106932 (cryptopia) - 0.06060000 (bleutrade) => 0.05953068 (5,567.15%)
ARBITRAGE FOUND: BLC/DOGE => 7.89999999 (cryptopia) - 420.00010012 (ccex) => 412.10010013 (5,216.46%)
ARBITRAGE FOUND: BLC/LTC => 0.00014999 (cryptopia) - 0.00780009 (ccex) => 0.00765010 (5,100.41%)
ARBITRAGE FOUND: XID/BTC => 0.00000130 (ccex) - 0.00004406 (liqui) => 0.00004276 (3,289.23%)
ARBITRAGE FOUND: BTG/DOGE => 7,943.29109999 (cryptopia) - 175,100.00000001 (bleutrade) => 167,156.70890002 (2,104.38%)
ARBITRAGE FOUND: FUEL/BTC => 0.00000069 (cryptopia) - 0.00001000 (hitbtc2) => 0.00000931 (1,349.28%)
ARBITRAGE FOUND: MTL/BTC => 0.00139941 (bittrex) - 0.02000006 (livecoin) => 0.01860065 (1,329.18%)
ARBITRAGE FOUND: IFC/LTC => 0.00000024 (bter) - 0.00000178 (cryptopia) => 0.00000154 (641.67%)
ARBITRAGE FOUND: QTUM/BTC => 0.00191100 (gateio) - 0.01100000 (livecoin) => 0.00908900 (475.61%)
ARBITRAGE FOUND: CAT/BTC => 0.00000223 (cryptopia) - 0.00000900 (hitbtc2) => 0.00000677 (303.59%)
```

I want to thank you once again for the great job. This library is really incredible.

**Comment by kroitor** [comment]
Hi! Thx for reporting this! As you may know, our goal is to unify symbols for consistency, and we will fix those inconsistencies you posted.

More here:
https://github.com/ccxt-dev/ccxt/wiki/Manual#naming-consistency
or here, for example: https://github.com/ccxt-dev/ccxt/issues?utf8=%E2%9C%93&q=yobit%20symbol%20

We will eliminate all of them in the end with your help. I'll update you as soon, as we fix the above. Thx once again!

**Comment by kroitor** [comment]
bx.in.th had reversed pairs in symbols, that is fixed in 1.9.244+

Bter has been shutdown by PBoC:

![screen shot 2017-10-24 at 20 19 09](https://user-images.githubusercontent.com/1294454/31957868-b708be5a-b8f8-11e7-997e-8475581b55b1.png)

**Comment by kroitor** [comment]
FUEL looks like an IOU token market at HitBTC with no real currency traded yet (that's a presale).

**Comment by kroitor** [comment]
This issue has been fixed as of version 1.9.245+. See the list of introduced changes above (or in commit history).

Also, these currencies were not changed:
FUEL, XID, MTL, QTUM. The discrepancies are real and if you take a closer look at them, you'll find out, that those are caused not by symbols being messed up, but by a lack of traded volumes, limitations of all sorts, ICO presale rules, etc.

Apart from that, all listed inconsistencies should have been corrected now. Thx again!
